src/app/Http/Controllers/QuestionController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Model\Question;
+use App\Model\Choice;
+use Illuminate\Http\Request;
+
+class QuestionController extends Controller
+{
+    //
+    public function index($id) {
+        $questions = Question::where('big_question_id', $id)->get();
```

Comment:
もし意図的に分けていないのであればwithを利用して一緒に持ってくると良いです
```suggestion
        $questionWithChoices = Question::with('choice')->where('big_question_id', $id)->get();
```